build_master_index

```diff
@@ -3,95 +3,79 @@
 #PBS -N build_cc_db
 #PBS -P v45
```

Comment:
We have to run this under a compute project, which means we're relying on the setgid bit for file outputs to be owned by ik11. Might be worth explicitly ensuring the DB and logs are ik11 in case something changes in the future?

Also good to add

#PBS -W umask=027
to ensure the PBS log and error files are group readable. Maybe is they're Jenkins artefacts this is a moot point? Not sure.

Reply:
Good point. I'll explicitly set the group to be ik11.

About the umask, this is not needed, because the log files get deleted after being saved as artifacts.

Reply:
The saving as artifacts take care of permissions? As in they viewable through Jenkins so doesn't matter?

Reply:
I don't think it matter as long as Jenkins is able to read the files. Since it's running under my account, this should be fine.